providers/tests/amazon/aws/log/test_s3_task_handler.py

```diff
@@ -128,7 +128,10 @@ def test_read(self):
         ti.state = TaskInstanceState.SUCCESS
         log, metadata = self.s3_task_handler.read(ti)
         actual = log[0][0][-1]
-        expected = "*** Found logs in s3:\n***   * s3://bucket/remote/log/location/1.log\nLog line"
+        expected = " INFO - ::group::Log message source details\n"
+        expected += "*** Found logs in s3:\n***   * s3://bucket/remote/log/location/1.log\n"
+        expected += " INFO - ::endgroup::\n"
+        expected += "Log line"
```

Comment:
Ah we had a bit of side-discussions about the exact match and... the Airflow CI now shows errors in back-compat tests.

Provider packages are tested for compatibility against old Airflow versions 2.10, 2.9 and 2.8. As the log grouping was not existing like this in the former Airflow-Versions, these tests fail...

So most probably, to make CI green you need to change the exact string match to an non-exact match like `assert "my expected text" in actual`
...then you are Airflow core version neutral.